rules/windows/process_creation/proc_creation_win_susp_parent_of_conhost.yml

```diff
@@ -0,0 +1,37 @@
+title: Conhost Spawned By Suspicious Parent Process
+id: cbb9e3d1-2386-4e59-912e-62f1484f7a89
+status: experimental
+description: Detects when the Console Window Host (conhost.exe) process is spawned by a suspicious parent process, which could be indicative of code injection.
+author: Tim Rauch
+references:
+    - https://www.elastic.co/guide/en/security/current/conhost-spawned-by-suspicious-parent-process.html
+date: 2022/09/28
+logsource:
+    category: process_creation
+    product: windows
+tags:
+    - attack.execution
+    - attack.t1059
+detection:
+    selection:
+        Image|endswith: '\conhost.exe'
+        ParentImage|endswith:
+            - '\svchost.exe'
```

Comment:
Known rare FP for svchost (at the moment) are

```yml
svchost.exe -k apphost -s AppHostSvc
svchost.exe -k imgsvc
```

Please include a filter excluding these two command lines